codecov/settings_base.py

```diff
@@ -92,13 +92,20 @@
 
 WSGI_APPLICATION = "codecov.wsgi.application"
 
-# Database
-# https://docs.djangoproject.com/en/2.1/ref/settings/#databases
+
+# GraphQL
 
 GRAPHQL_QUERY_COST_THRESHOLD = get_config(
     "setup", "graphql", "query_cost_threshold", default=10000
 )
 
+GRAPHQL_RATE_LIMIT_VALUE = get_config(
```

Comment:
Thoughts on a name that includes the units? Like GRAPHQL_RATE_LIMIT_RPM or similar?

Reply:
Oooo I like that, I was straining my brain thinking of a name besides "Value" lol